ain/ain.py

```diff
@@ -0,0 +1,40 @@
+from ain.provider import Provider
+from ain.net import Network
+from ain.wallet import Wallet
+from ain.db import Database
+from ain.utils import ecSignTransaction, getTimestamp
+
+
+class Ain:
+    def __init__(self, providerUrl: str, chainId: int = 0):
+        self.provider = Provider(self, providerUrl)
+        self.chainId = chainId
+        self.net = Network(self.provider)
+        self.wallet = Wallet(self, self.chainId)
+        self.db = Database(self, self.provider)
+
+    # TODO : specify type `TransactionInput`
```

Comment:
Please try to follow our TODO style: https://www.notion.so/comcom/ComCom-Developer-s-Guide-ad57ebc31ef94040a4f6e71d9c2042c5

Reply:
Done